core/domain/exp_domain.py

```diff
@@ -1269,7 +1274,8 @@ def __init__(self, content, param_changes, interaction,
             interaction.hints, interaction.solution)
         self.classifier_model_id = classifier_model_id
 
-    def validate(self, exp_param_specs_dict, allow_null_interaction):
+    def validate(self, exp_param_specs_dict, allow_null_interaction,
```

Comment:
Nit: Break after the parenthesis '('. This should fix the lint failure.

Reply:
Done.